app/views/petitions/show.html.erb

```diff
@@ -12,69 +12,62 @@
   </div>
 </details>
 
-<%= content_tag :p, :class => 'flash-notice' do -%>
-  This e-petition is now closed
-<% end if @petition.closed? %>
+<% if @petition.rejected? %>
+  <%= render 'rejected_petition' %>
+<% else %>
 
-<h1>
-  <span class="heading-secondary">Petition</span>
-  <%= @petition.title %>
-</h1>
+  <%= content_tag :p, :class => 'flash-notice' do -%>
```

Comment:
Nope. I meant that we have a partial which represents a rejected petition. Maybe we should have a partial that represents open or not-rejected. It's more of a question to see what others think.